optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -248,8 +248,7 @@ def _render(self, result, output_path):
             else:
                 from pystache import view
                 v = view.View(template, context)
-                report_contents = v.render('utf8')  # encodes into ascii
-                report_contents = codecs.decode("utf8")  # decode to unicode
+                report_contents = v.render('utf8')
```

Comment:
@apahim I don't think so, because there's no magical way to fix pystache at that version, as it's explained in the commit message.  We could rewrite parts of the pystache code and use those overloaded classes, but, IMHO, this is as much as we should try to do for now.  @ldoktor do you agree or have other ideas here?